modules/datastore/src/Storage/DatabaseTable.php

```diff
@@ -158,6 +159,19 @@ public function setSchema($schema) {
     parent::setSchema($schema);
   }
 
+  /**
+   * Disable/enable InnoDB strict mode for the given database connection.
+   *
+   * @param string $value
+   *   "ON" or "OFF".
+   */
+  public function setInnodbMode(string $value = "OFF"): void {
+    // Only if we're using MySQL.
+    if ($this->connection instanceof MysqlConnection) {
+      $this->connection->query("SET SESSION innodb_strict_mode=:value", [':value' => $value]);
```

Comment:
This method will throw a database exception if `$value` is not "ON" or "OFF". Could we make the `$value` parameter a boolean and rename the function appropriately, or just add a check to make sure `$value` is either "ON" or "OFF"?

Reply:
Yeah good point, I was rushing here. Will do.

Reply:
:raised_hands: Awesome! Love it!